src/components/Pressable.tsx

```diff
@@ -0,0 +1,281 @@
+import React, { useRef } from 'react';
+import {
+  View,
+  AccessibilityProps,
+  ColorValue,
+  Insets,
+  NativeSyntheticEvent,
+  StyleProp,
+  TargetedEvent,
+  ViewProps,
+  ViewStyle,
+  StyleSheet,
+} from 'react-native';
+
+import {
+  Gesture,
+  GestureDetector,
+  GestureHandlerRootView,
+  GestureStateChangeEvent,
+  GestureTouchEvent,
+  LongPressGestureHandlerEventPayload,
+  TouchData,
+} from '../.';
```

Comment:
Now that I think about it, it shouldn't be imported this way as it creates a cycle